src/pages/AboutUs.tsx

```diff
@@ -0,0 +1,112 @@
+import React from "react";
+import NavBar from "../components/navigation/NavBar";
+import { makeStyles, Typography, Link } from "@material-ui/core";
+
+import backgroundImage from  "../media/about_us_background.png"
+const useStyles = makeStyles(theme => ({
+  wrapper: { 
+    background: `url(${backgroundImage}) no-repeat center center`,
+    backgroundSize: "cover"
+  },
+  container: {
+    background: "rgba(255, 255, 255, 0.6)",
+    padding:"2rem",
+    [theme.breakpoints.up("sm")]:{
+      padding:"2rem 7rem",
+    },
+   
+  },
+  overlapContainer: {
+    background: "rgba(255, 255, 255, 0.8)",
+    padding:"0 2rem",
+    [theme.breakpoints.up("sm")]:{
+      padding:"0 7rem",
+    },
+  },
+  paragraphContainer: {
+   padding: "1rem 0"
+  },
+  boldText: {
+    fontWeight: "bold"
+  },
+  bottomContainer: {
+    backgroundColor: "white",
+    padding: "2rem",
+    [theme.breakpoints.up("sm")]:{
+      padding:"2rem 7rem",
+    },
+  },
+  titleWrapper:{
+    paddingTop: "2rem",
+    [theme.breakpoints.up("sm")]: {
+      paddingTop: "4rem"
+    },
+  }
+}));
+
+function AboutUs() {
+  const classes = useStyles();
+  return (
+    <>
+      <NavBar showLogout />
+      <div className={classes.wrapper}>
+        <div className={classes.container}>
+          <div className={classes.titleWrapper}>
+            <Typography variant="h3" component="h1" align="center" color="primary">About Us</Typography>
+          </div>
+
+          <Typography variant="body1" component="p" align="center" color="secondary" className={`${classes.paragraphContainer} ${classes.boldText}`}>
```

Comment:
Seems that those properties are repeated a lot for all for the next Typographie - we can factorise them into a single object to avoid redundancy :)